chromium_src/chrome/browser/media/router/media_router_feature.cc

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/media/router/media_router_feature.h"
+
+#include "chrome/common/pref_names.h"
+#include "extensions/common/feature_switch.h"
+
+#define MediaRouterEnabled MediaRouterEnabled_ChromiumImpl
+#include "../../../../../../chrome/browser/media/router/media_router_feature.cc"  // NOLINT
+#undef MediaRouterEnabled
+
+namespace media_router {
+
+// Media router pref can be toggled using kLoadMediaRouterComponentExtension
+// on Desktop
+void UpdateMediaRouterPref(content::BrowserContext* context) {
+#if BUILDFLAG(ENABLE_EXTENSIONS)
+  user_prefs::UserPrefs::Get(context)->SetBoolean(
```

Comment:
`GetMediaRouterPref(context)->SetBoolean` ? Seems odd to use `user_prefs::UserPrefs` here and `GetMediaRouterPref` here https://github.com/brave/brave-core/pull/2836/files#diff-5bcfa4281cb261b90bf7b2479f5c461dR31